src/nvim/ex_cmds.c

```diff
@@ -6035,10 +6034,7 @@ FUNC_ATTR_NONNULL_ARG(1, 2, 3)
     }
 
     // computing the length of the column that will display line number
-    //TODO: Is log10 efficient for counting the nr of digits?
-    int col_width = log10(highest_num_line) + 1 + 3;
-    // better? log10 might not be precise for large numbers (ref?)
-    //int col_width = snprintf(NULL, 0, "%ld", highest_num_line) + 3;
+    int col_width = snprintf(NULL, 0, "%ld", highest_num_line) + 3;
```

Comment:
I still think `log10` is the right choice instead of snprintf.  If it were more than just figuring out the width of the number, then snprintf would make sense, but using a printf-family function for this seems like overkill.